src/btree/bt_curnext.c

```diff
@@ -756,26 +756,7 @@ __wt_btcur_next_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
     need_walk = false;
     session = CUR2S(cbt);
     total_skipped = 0;
-    /*
-     * If we have a bound set we should position our cursor appropriately if it isn't already
-     * positioned.
-     *
-     * In one scenario this function returns and needs to walk to the next record. In that case we
-     * set a boolean. We need to be careful that the entry flag isn't set so we don't re-enter
-     * search near.
-     *
-     * This logic must come before the call to __wt_cursor_func_init, as it will set the cbt active
-     * flag which changes the logic flow within the subsequent search near resulting in a
-     * segmentation fault. It is better to not set state prior to calling search near. Additionally
-     * the cursor initialization function does take a snapshot if required as does cursor search
-     * near.
-     */
-    if (F_ISSET(cursor, WT_CURSTD_BOUND_LOWER) && !WT_CURSOR_IS_POSITIONED(cbt) &&
-      !F_ISSET(cursor, WT_CURSTD_BOUND_ENTRY)) {
-        WT_ERR(__wt_btcur_bounds_row_position(session, cbt, true, &need_walk));
-        if (!need_walk)
-            return (0);
-    }
+
```

Comment:
delete newline.

Reply:
Previously before the bounds implementation there was a space here. I think we should add stats differently from the initialisation of the variables.